### PR TITLE
Removes inconsistencies between the definitions of the set functions

### PR DESCRIPTION
**Pull request by hartig**
... as pointed out in #99

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-query/pull/108.html" title="Last updated on Jun 29, 2023, 10:56 AM UTC (dedb4d0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-query/108/ae554d5...dedb4d0.html" title="Last updated on Jun 29, 2023, 10:56 AM UTC (dedb4d0)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -9537,6 +9538,11 @@ <h4>Aggregate Algebra</h4>
             <p>The Flatten(S) function takes a sequence of lists, S = [(L<sub>1</sub>, L<sub>2</sub>,
               ...), ...], and returns the list ( x | L in S and x in L ).</p>
           </div>
+          <p>Card is a function that returns the cardinality of a sequence or a list of elements (which may be solution mappings or other types of elements, depending on the context).
+          <div class="defn">
+            <p><b>Definition: <span id="defn_Card">Card</span></b></p>
+            <p>Given a sequence or a list |L|, Card(|L|) is the cardinality of |L|.</p>
```
Comment (rubensworks) [inline spec/index.html:9544]: Should the list be named just `L`, with `|L|` being syntatic sugar for `Card(L)`?

```suggestion
            <p>Given a sequence or a list L, Card(L) is the cardinality of L.</p>
```

Comment (kasei) [reply spec/index.html:9544]: That's [respec shorthand markup for a variable](https://respec.org/docs/#variable-shorthands).

Comment (rubensworks) [reply spec/index.html:9544]: Ah, TIL, thanks for letting me know!

**Review by rubensworks — APPROVED** [review]

**Review by Tpt — APPROVED** [review]

**Comment by afs** [comment]
`card[ ]` appears in other places as well.

There's a definition of the notation "Write card[Ω](μ) for the cardinality of solution mapping μ in a multiset of mappings Ω." in 18.3

If they are the same, then they need to be aligned.
If they are different and `card(` is specific to aggregation usage, then maybe we need a different way to write it.

**Comment by hartig** [comment]
I am aware of `card[Ω](μ)` as used in other parts of the spec. That function has two arguments---a multiset Ω of solution mappings and a solution mapping μ---and, as you quote, it denotes how often the solution mapping μ occurs within Ω. Before this PR, a similar notation was used also for a different purpose: That is, `card[L]` was used to denote the cardinality (the size) of some list/sequence `L`. But this notation was not defined anywhere in the spec. This PR fixes this issue by explicitly defining a function called Card. Notice also that, in all the places in which I use this function, the function name is linked to the definition. We may do the same for the notation `card[Ω](μ)`.

**Comment by afs** [comment]
`card[Ω]` evaluates to a function that gives the number of instances of `μ`.

We could change that "multiplicity" or something based on that word ("multiplicity" seems a bit long but may be necessary).

**Comment by hartig** [comment]
> We could change that "multiplicity" or something based on that word ("multiplicity" seems a bit long but may be necessary).

I agree. Yet, I think this is a separate question (separate from this PR here). I have created PR #110 with a proposal of how I think we may replace the notation `card[Ω](μ)` by a function called 'multiplicity' that has a definition that can be cross-referenced.

**Comment by hartig** [comment]
@afs Do you agree that the discussion of whether and how the notation `card[Ω](μ)` should be changed is independent of the changes in this PR and may be taken elsewhere instead (e.g., to PR #110 in which I have proposed such a change for `card[Ω](μ)`)?

If you agree, are there any other changes in this PR that you see as potentially problematic?

Otherwise, I may still merge this PR before going on vacation.

**Review by afs — APPROVED** [review]

**Comment by afs** [comment]
Yes, I agree. Merging before your break is a good idea.

**Pull request merged**